main.js

```diff
@@ -19,4 +11,19 @@ setInterval(function () {
         input.removeAttribute('disabled');
     }
 }, 2000);
-_queryS('#queue-btns .btn-group').appendChild(mybt);
\ No newline at end of file
+
+var _chatButtonId = 'naugtur-chat',
+    _chatButtonMissing = !_queryS('#' + _chatButtonId);
```

Comment:
:+1: I see what you did there 
Now it supports people who like running it multiple times for no reason :)